helmfile.d/upstream/opensearch-project/opensearch/ci/ci-values.yaml

```diff
@@ -97,7 +97,9 @@ config:
 # Extra environment variables to append to this nodeGroup
 # This will be appended to the current 'env:' key. You can use any of the kubernetes env
 # syntax here
-extraEnvs: []
+extraEnvs:
+  - name: OPENSEARCH_INITIAL_ADMIN_PASSWORD
+    value: myStrongPassword123@456
```

Comment:
So this is not in effect because we configure the securityplugin and the admin user [here](https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile.d/values/opensearch/securityadmin.yaml.gotmpl#L49-L50)?